Missing environment variables [LIBIIO_INCLUDEDIR]

When I clone the code,
mkdir build / cd build / cmake ..
an error occurred

Make Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
LIBIIO_INCLUDEDIR
CMake Error in CMakeLists.txt:
Found relative path while evaluating include directories of "ad9361":
"LIBIIO_INCLUDEDIR-NOTFOUND"

I think libad9361 requires libiio dependency, and this operation is not mentioned in the Readme.md. When I added my libiio path "set(LIBIIO_INCLUDEDIR "/home/XX/drivers/libiio/include")" in CMakeLists.txt , this problem is solved

So could you add some additional explanations to the Readme.md?

--- a/README.md
+++ b/README.md
@@ -3,7 +3,11 @@
 This is a simple library used for userspace,
  - which manages multi-chip sync (on platforms (FMCOMMS5) where multiple AD9361 devices are use)
  - can create AD9361 specific FIR filters on the fly,
- 
+
+**Dependencies**
+
+ - [libiio IIO interface library (Version less than v1.0)](https://github.com/analogdevicesinc/libiio)
+
 **Docs**
 
 Doxygen-based documentation is available at: http://analogdevicesinc.github.io/libad9361-iio/

--- a/bindings/python/README.md
+++ b/bindings/python/README.md
@@ -11,7 +11,7 @@ If you have a question about libad9361-iio or the AD9361 transceiver please ask 
 
 ## Requirements
 To use these bindings naturally you need the core library they depend upon, libad9361-iio. This is not packaged with the pypi release but there are a number of options:
-  - If you want to just use libad9361-iio, we suggest using the [latest release](https://github.com/analogdevicesinc/libad9361-iio/releases/latest).
+  - If you want to just use libad9361-iio, we suggest using the [latest v0.XX release](https://github.com/analogdevicesinc/libad9361-iio/releases/latest) (v1.XX is not supported yet).
   - If you think you have found a bug in the release, or need a feature which isn't in the release, try the latest **untested** binaries from the master branch and check out the [documentation](https://codedocs.xyz/analogdevicesinc/libad9361-iio/) based on the master branch. We provide builds for a few operating systems. If you need something else, we can most likely add that -- just ask.
 
 ### Installing the bindings

--- a/mainpage.dox
+++ b/mainpage.dox
@@ -18,6 +18,8 @@ This is a simple library used for userspace management control and configuration
  - which manages multi-chip sync (on platforms (FMCOMMS5) where multiple AD9361 devices are use)
  - can create AD9361 specific FIR filters on the fly
 
+The library requires:
+  - libiio version 0.X (>= 1.0.0 is not supported yet): http://analogdevicesinc.github.io/libiio/
 
 @section ad9361_hardware AD9361 Transceiver Internals
 

